client/src/Modules/General/ButtonComponent.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {Button} from 'react-bootstrap';
 
 
 const ButtonComponent = ({ type , name, dataid = null, id, value ='Button' ,className, onClickfunction }) => {
    
-    return <Button type={type} name={name} dataid={dataid} id={id} className = {className} onClick={onClickfunction}>{value}</Button>;
+    return <button type={type} name={name} dataid={dataid} id={id} className = {className} onClick={onClickfunction}>{value}</button>;
 };
 
 ButtonComponent.propTypes = {
```

Comment:
I like PropTypes.

I think it's fine to introduce them gradually now as we're still figuring out the app, but as long as we have the understanding that `PropTypes` will be used throughout.